WooCommerce/Classes/POS/Presentation/Reusable Views/POSSendReceiptModalView.swift

```diff
@@ -0,0 +1,84 @@
+import SwiftUI
+
+struct POSSendReceiptModalView: View {
+    @EnvironmentObject private var posModel: PointOfSaleAggregateModel
+    @Environment(\.dismiss) private var dismiss
+
+    @State private var textFieldInput: String = ""
+
+    var body: some View {
+        ZStack(alignment: .topTrailing) {
+            VStack(alignment: .leading, spacing: 20) {
+                Text(Localization.title)
+                    .font(.largeTitle)
+                    .bold()
+
+                Text(Localization.subtitle)
+                    .font(.headline)
+
+                TextField(Localization.textfieldPlaceholder, text: $textFieldInput)
+                    .keyboardType(.emailAddress)
+                    .textInputAutocapitalization(.none)
+                    .autocorrectionDisabled()
+                    .textFieldStyle(RoundedBorderTextFieldStyle(focused: true))
+                    .padding(.horizontal)
+
+                Button(action: {
+                    Task { @MainActor in
+                        await posModel.sendReceipt(to: textFieldInput)
+                    }
+                }, label: {
+                    HStack(spacing: Constants.buttonSpacing) {
+                        Text(Localization.buttonTitle)
+                            .font(Constants.buttonFont)
+                    }
+                    .frame(minWidth: UIScreen.main.bounds.width / 2)
+                })
+                .padding(Constants.buttonPadding)
+                .foregroundColor(Color.posPrimaryTextInverted)
+                .background(Color.posOverlayFillInverted)
+                .cornerRadius(Constants.buttonCornerRadius)
+
+                Spacer()
+            }
+            .padding(.top)
+            .padding()
+
+            Button(action: {
+                dismiss()
+            }) {
+                Image(systemName: "xmark")
+                    .foregroundColor(.gray)
+                    .font(.title2)
+                    .padding()
+            }
```

Comment:
This, along with other stuff, could be removed by using a POSModal, and it would probably look more consistent with the rest of POS. You can still present it from the TotalsView with the same binding.

Reply:
True, I made the change to use `.posModal` on https://github.com/woocommerce/woocommerce-ios/pull/14481/commits/9547ed82bc4d8487fdf9819e99ef20afda3cf68e